benchmark/initializer.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile do
+  source "https://rubygems.org"
+  gem "dry-initializer", "~> 3.1"
+  gem "dry-types", "~> 1.5"
+  gem "awesome_print"
+  gem "benchmark-ips"
+  gem "pry"
+  gem "ruby-prof"
+  gem "sorbet"
+  gem "sorbet-runtime"
+  gem "definition", path: File.expand_path("../.", __dir__)
+end
+
+class SorbetUseCase
+  extend T::Sig
+
+  sig { params(phone: String, admin: T::Boolean).void }
```

Comment:
Honestly this might also be a very good way to do validation in our service usecase classes. It required more verbose code though, as you need to specify the initializer function with all params, then assign all values and provide attr_accessors yourself

Reply:
I liked to have the params types in an explicit way!

Reply:
it looks really ugly if you have 6 parameters for example :/

Reply:
continuing with getting this merged and used now as it is our preferred solution for now